src/components/Blog/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import { Row, Col, Container, Image } from "react-bootstrap";
+import "./index.css";
+
+const Blog = (props) => {
+  const storyID = props.match.params.id;
```

Comment:
I dont think this is done in the correct way, there should be something called useParams that u get the id from please check this page for further info https://reactrouter.com/web/example/url-params

Reply:
So I tried using the same method, I succeeded in loading the page with the id. However, when I'm using "dangerouslySetInnerHTML", the page won't load as the fitch takes some time and it takes them as undefined.
how do you suggest fixing this issue? @halitbatur